tests/Operation/RenameCollectionFunctionalTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Model\CollectionInfo;
+use MongoDB\Model\CollectionInfoIterator;
+use MongoDB\Operation\InsertOne;
+use MongoDB\Operation\ListCollections;
+use MongoDB\Operation\RenameCollection;
+use MongoDB\Tests\CommandObserver;
+use function sprintf;
+use function version_compare;
+
+class RenameCollectionFunctionalTest extends FunctionalTestCase
+{
+    public function testDefaultWriteConcernIsOmitted()
+    {
+        (new CommandObserver())->observe(
+            function () {
+                $operation = new RenameCollection(
+                    'admin',
```

Comment:
I was originally using `$this->getDatabaseName()` for the `$databaseName` parameter, but it gave the error `MongoDB\Driver\Exception\CommandException: renameCollection may only be run against the admin database` so I changed all the tests to use the `'admin'` database - not sure if that was the right thing to do.

Reply:
The [server manual]((https://docs.mongodb.com/manual/reference/command/renameCollection/) does state that the command itself must be run against the `admin` database:

> Issue the renameCollection command against the admin database.

However, that shouldn't limit the source and target namespaces involved in the renaming.